src/ansys/openapi/common/_oidc.py

```diff
@@ -33,10 +30,6 @@ class OIDCSessionFactory:
     Creates an OpenID Connect session with the configuration fetched from the API server. This class uses either
     the provided token credentials or authorizes a user with a browser-based interactive prompt.
 
-    If your identity provider does not provide the exact scopes requested by your API server, you will be unable to
```

Comment:
Does this functionality still exist? If not, do you think it's likely to be a significant regression?

Reply:
I think if it's outside the spec and it works, we probably don't need to test it.